Lib/fontbakery/profiles/ufo_sources.py

```diff
@@ -19,22 +16,26 @@
 
 
 @condition
-def ufo_font(ufo):
+def ufo_font(font):
     try:
         import defcon
-        return defcon.Font(ufo)
-    except:
+        return defcon.Font(font)
+    except Exception:
         return None
 
 
 @check(
     id = 'com.daltonmaag/check/ufolint',
     proposal = 'https://github.com/googlefonts/fontbakery/pull/1736'
 )
-def com_daltonmaag_check_ufolint(ufo):
```

Comment:
A `font` is a font binary path (as opposed to a `ttFont` being a fontTools' TTFont object) and an `ufo` is a font project in UFO format.

Because of that, I believe all of these changes on `ufo_sources.py` are incorrect.

Reply:
Yeah, I understand the concept, but there are parts of the infrastructure that don't seem to abide with that convention. What I can tell you is that all of these changes are needed to make the tests success; but perhaps that's only because of bugs in other parts of the maze 😵‍💫
I need to take another look.

Reply:
yes, I agree that some parts of the codebase may feel a bit like a maze, indeed.